package.json

```diff
@@ -33,7 +33,7 @@
   "engineStrict" : true,
   "dependencies": {
     "@interlay/esplora-btc-api": "^0.3.3",
-    "@interlay/polkabtc-types": "0.2.9",
+    "@interlay/polkabtc-types": "0.2.12",
```

Comment:
This should still be the 0.2.9 version I assume. 0.2.12 has the latest FixedPoint additions in there that we don't want on master yet.